doc/viewers.rst

```diff
@@ -16,15 +16,15 @@ spikeinterface.widgets
 
 The easiest way to visualize :code:`spikeinterface` objects is to use the :code:`widgets` module for plotting.
 You can find an extensive description in the module documentation :ref:`modulewidgets`
-and many examples in this tutorial :ref:`sphx_glr_modules_gallery_widgets`.
```

Comment:
For context, this was generating a missing reference and I couldn't find a way to ref a dynamically generated sphinx-gallery module, but just individual files.

Reply:
A lot of the sphinx gallery stuff is annoying like that, but based on my reading they don't really have plans to try to patch things like that. Such is life I suppose.